authentik/lib/config.py

```diff
@@ -188,19 +252,12 @@ def update_from_env(self):
             if not key.startswith(ENV_PREFIX):
                 continue
             relative_key = key.replace(f"{ENV_PREFIX}_", "", 1).replace("__", ".").lower()
-            # Recursively convert path from a.b.c into outer[a][b][c]
-            current_obj = outer
-            dot_parts = relative_key.split(".")
-            for dot_part in dot_parts[:-1]:
-                if dot_part not in current_obj:
-                    current_obj[dot_part] = {}
-                current_obj = current_obj[dot_part]
             # Check if the value is json, and try to load it
             try:
                 value = loads(value)
```

Comment:
Could we try to parse and load b64 here instead of explicitly having to call `get_dict_from_b64_json`? Maybe not such a good idea since some values might be b64 encoded that shouldn't be decoded?

Reply:
Furthermore, b64 encoded values can then only be set in env vars and not in the yaml files.